packages/api/internal/handlers/sandbox_get.go

```diff
@@ -95,6 +108,7 @@ func (a *APIStore) GetSandboxesSandboxID(c *gin.Context, id string) {
 		State:           api.Paused,
 		EnvdVersion:     lastSnapshot.EnvBuild.EnvdVersion,
 		EnvdAccessToken: sbxAccessToken,
+		Domain:          nil,
```

Comment:
why is it nil in here?

Reply:
Is it correct to return the domain for a sandbox that does not run, so you technically don't know what URL you should return? Looks to me the same as previously, did not return client ID here, because in this state, you don't know on what node the sandbox was running. 

If we decide that we want to always return the domain, then yes, we don't have any other way than to return the cluster domain/default domain here.